pipelines/binary-builder.yml

```diff
@@ -1,8 +1,8 @@
+<% auto_build_names = %w(bower composer glide godep nginx node yarn) %>
```

Comment:
This _may_ also need the `dotnet` component if we resurrect the `dotnet-framework` buildpack.